prowler/providers/aws/services/elasticbeanstalk/elasticbeanstalk_service.py

```diff
@@ -0,0 +1,87 @@
+    def __init__(self, provider):
+        # Call AWSService's __init__
+        super().__init__(__class__.__name__, provider)
+        self.environments = {}
+        self.__threading_call__(self._describe_environments)
+        self.__threading_call__(
+            self._describe_configuration_settings, self.environments.values()
+        )
+
+    def _describe_environments(self, regional_client):
+        logger.info("ElasticBeanstalk - Describing environments...")
+        try:
+            describe_environment_paginator = regional_client.get_paginator(
+                "describe_environments"
+            )
+            for page in describe_environment_paginator.paginate():
+                for environment in page["Environments"]:
+                    environment_arn = environment["EnvironmentArn"]
+                    if not self.audit_resources or (
+                        is_resource_filtered(environment_arn, self.audit_resources)
+                    ):
+                        self.environments[environment_arn] = Environment(
+                            id=environment["EnvironmentId"],
+                            arn=environment_arn,
+                            application_name=environment["ApplicationName"],
+                            name=environment["EnvironmentName"],
+                            region=regional_client.region,
+                        )
+        except Exception as error:
+            logger.error(
+                f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
+            )
+
+    def _describe_configuration_settings(self, environment):
+        logger.info("ElasticBeanstalk - Describing configuration settings...")
+        try:
+            regional_client = self.regional_clients[environment.region]
+            configuration_settings = regional_client.describe_configuration_settings(
+                ApplicationName=environment.application_name,
+                EnvironmentName=environment.name,
+            )
+            option_settings = configuration_settings["ConfigurationSettings"][0].get(
+                "OptionSettings", {}
+            )
+            for option in option_settings:
+                if (
+                    option["Namespace"] == "aws:elasticbeanstalk:healthreporting:system"
+                    and option["OptionName"] == "SystemType"
+                ):
+                    environment.health_option = option
+                elif (
+                    option["Namespace"] == "aws:elasticbeanstalk:managedactions"
+                    and option["OptionName"] == "ManagedActionsEnabled"
+                ):
+                    environment.managed_actions_option = option
+                elif (
+                    option["Namespace"] == "aws:elasticbeanstalk:cloudwatch:logs"
+                    and option["OptionName"] == "StreamLogs"
+                ):
+                    environment.cloudwatch_option = option
```

Comment:
Why we don't store only the value of the option?

Reply:
Thought about it at first. Added here [7f64d1f](https://github.com/prowler-cloud/prowler/pull/5322/commits/7f64d1fe6669249b70116abd73c310335c4cda4d)